pkg/mocks/drivemgr.go

```diff
@@ -48,18 +56,23 @@ func NewMockDriveMgrClient(drives []*api.Drive) *MockDriveMgrClient {
 }
 
 // SetDrives set drives for current MockDriveMgrClient instance
-func (m MockDriveMgrClient) SetDrives(drives []*api.Drive) {
+func (m *MockDriveMgrClient) SetDrives(drives []*api.Drive) {
 	m.drives = drives
 }
 
 // AddDrives extends drives slice
-func (m MockDriveMgrClient) AddDrives(drives ...*api.Drive) {
+func (m *MockDriveMgrClient) AddDrives(drives ...*api.Drive) {
 	m.drives = append(m.drives, drives...)
 }
 
 // GetDrivesList returns provided to MockDriveMgrClient drives to imitate working of DriveManager
-func (m MockDriveMgrClient) GetDrivesList(ctx context.Context, in *api.DrivesRequest, opts ...grpc.CallOption) (*api.DrivesResponse, error) {
+func (m *MockDriveMgrClient) GetDrivesList(ctx context.Context, in *api.DrivesRequest, opts ...grpc.CallOption) (*api.DrivesResponse, error) {
 	return &api.DrivesResponse{
 		Disks: m.drives,
 	}, nil
 }
+
+// Locate is a stub for Locate DriveManager's method
+func (m MockDriveMgrClient) Locate(ctx context.Context, in *api.DriveLocateRequest, opts ...grpc.CallOption) (*api.DriveLocateResponse, error) {
```

Comment:
You've made every method by a pointer and left this one by value. I think it's a good idea to make it in the same way

Reply:
done